src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -336,7 +339,9 @@ const ResourceList = () => {
                     fixedTabComponents.map((component, index) => {
                         /* NOTE: need to retain terminal layout. Thus hiding it through visibility */
                         const hideClassName =
-                            tabs[index].name === AppDetailsTabs.terminal ? 'dc__visibility-hidden' : 'dc__hide-section'
+                            tabs[index].name === AppDetailsTabs.terminal
+                                ? 'cluster-terminal-hidden'
```

Comment:
The reason I had removed the showTerminal check is because since I am showing/hiding divs from resourceList why pass a prop for this. Thus brought the css to ResourceBrowser.scss and using it here. In hindsight I should have checked what was inside this css. Will be more careful henceforth.